src/hooks/swap/useSendSwapTransaction.tsx

```diff
@@ -36,6 +36,7 @@ export default function useSendSwapTransaction(
         callback: async function onSwap(): Promise<ExecTxResponse> {
           const approveTx = await approve({
             chainId: trade.fromToken.chainId as ChainId,
+            toChainId: trade.toToken.chainId,
             fromToken: trade.fromToken.wrapped.address,
             toToken: trade.fromToken.wrapped.address,
```

Comment:
why both these are `fromToken` tho?

Reply:
It is the same question that I asked. i would assume it was a mistake, let's make it toToken